### ISSUE TITLE
Macross Trial Frontier  [BLJS93001]

### ISSUE THREAD

**Issue by vlahka1** [RPCS3/rpcs3#9842]
## Quick summary

Macross Trial Frontier does not auto save

## Details

Like all the Macross games, when you complete a stage it gives you a score, which shows up on the side of that stage that you can select.  At this point its auto saved.  If you reload the game back up the score will still be there and you can advance.  For some reason the game is not autosaving.  The score will show up, I can quit to start menu and then enter in again and find the score still there.  But if I close the game and then reload, the progress has been wiped and it shows nothing completed.  This means I cant advance in the game as it only shows 3 stages at a time.  Without completing one, the rest will not unlock.  Its impossible to complete.  I have used previous emulator versions from 5 months back to 2 years ago.  All of them do the same thing.  

The game also has a tendancy to freeze during gameplay.   Sometimes during the first stage, but uusally it takes 3-4 stages before it happens.  I once went 7 stages before it froze.  Changing settings doesnt seem to help much.  This isnt really an issue as you can just restart that stage.  But since its not saving my progress, this is not an option.  So its not even possible to leave the game on and try to power through to the end without turning it off.  

There seem to be no graphic or sound anomalies that I have noticed.  Its very solid.

[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/6038660/RPCS3.log.gz)

#### 5. Please provide your system configuration:

    OS - Win10
    CPU- i7 8700K
    GPU- nVidia 1660
    Driver version - 460.79
    16GB Ram

**Comment by Megamouse** [comment]
This may be fixed now.
In the past, we simply killed the game when we closed it.
But now, we tell the game to kill itself (lol).
This means, games can now properly close their resources, save their data, etc....

**Comment by vlahka1** [comment]
> This may be fixed now. In the past, we simply killed the game when we closed it. But now, we tell the game to kill itself (lol). This means, games can now properly close their resources, save their data, etc.... before terminating.

Just updated the emulator to its newest  revision.  Sadly its still not saving.

**Comment by vlahka1** [comment]
Does anyone know if an autosave would not work because of a placement of the game in the wrong folder, or a permission on the folders like read only etc.  I've tried a different game that had autosave that worked, but all 3 macross games refuse to keep their saved state when reloaded.  Would be so much easier if this emulator had a save state option.

**Comment by FlexBy420** [comment]
Is this still an issue?

**Comment by digant73** [comment]
@vlahka1 is that still an issue?

**Comment by vlahka1** [comment]
> [@vlahka1](https://github.com/vlahka1) is that still an issue?

Yes unfortunately.  At least the easy work around is to use the save state option in the emulator.  Not ideal but works.  Sadly the 30th anniversary macross is still absolutely broken.  Probably always will be.

**Comment by JBapt23** [comment]
Still as an issue in all macross games

**Comment by vlahka1** [comment]
At least we can use save state to save progress and get around this bug.  I'm resigned to seeing it never fixed.  At least it loads unlike the 30th anniversary game which is just an absolute mess.
